confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -269,7 +270,8 @@ def test_compatibility(self, subject, avro_schema, version='latest'):
             else:
                 log.error("Unable to check the compatibility")
                 False
-        except:
+        except: # noqa
```

Comment:
Shouldn't you at least restrict this to `except Exception:`?

Reply:
That's what I thought too, until I found this:

> In situations where you need to catch all “normal” errors, such as in a framework that runs callbacks, you can catch the base class for all normal exceptions, Exception. Unfortunately in Python 2.x it is possible for third-party code to raise exceptions that do not inherit from Exception, so in Python 2.x there are some cases where you may have to use a bare except: and manually re-raise the exceptions you don’t want to catch.

from https://docs.python.org/2/howto/doanddont.html#except

Reply:
Are the exceptions being caught here from the app code or from this library (and other open-source libraries)?

If app code, that makes sense, but if from libraries, than I don't think it makes sense as generally most open source libs are well-behaved and if it is an issue, maintainers are generally very open to cleaning this up.